.github/workflows/publish.yaml

```diff
@@ -0,0 +1,30 @@
+name: Release Package
+
+# need to setup:
+# secrets.PYPI_API_TOKEN (See: https://pypi.org/help/#apitoken)
+# secrets.TESTPYPI_API_TOKEN (See: https://test.pypi.org/help/#apitoken)
```

Comment:
> [repo secrets](https://docs.github.com/en/actions/security-for-github-actions/security-guides/using-secrets-in-github-actions#creating-secrets-for-a-repository) (`TESTPYPI_API_TOKEN`, `PYPI_API_TOKEN`) needed

I do not have the authority to set this up.
@wing328 If I send these to DM via Slack, can you set it up?

Reply:
sure

Reply:
Sent. Please check!

Reply:
done

Reply:
To get environment secrets its name are required. So please tell me it.
usage:　https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#example-using-a-single-environment-name